README.md

```diff
@@ -0,0 +1,61 @@
+# About workshop
+
+In this workshop we will explore what Firebase Functions are and setup project to be able to write those functions in Kotlin. To get there we will touch some other interesting things like Realtime Database and TypeScript. 
+
+
+### Requirements:
+
+ * [Firebase account](https://console.firebase.google.com/)
+ * Code editor for JavaScript/TypeScript (my choice is [VS Code](https://code.visualstudio.com/))
+ * [IntelliJ Idea community edition](https://www.jetbrains.com/idea/download/)
+
+
+### What are we building:
+
+> **Ultimate Currency converter**
+> 
+> Let's make a backend for the ultimate currency convertion app - user enters price in euro and backend converts it to other currencies.
+
+Backend will work as follows: 
+ * Client app pushes new "conversion" with euro value into table
+ * Function gets triggered and picks up this value and calculates other currencies
+ * Data gets written back to same object with a timestamp
+
+Firebase Realtime Database structure:
+``` js
+{
+    "rates" : {
+        "usd" : 1.175,
+        "czk" : 25.88,
+        "pln" : 4.303,
+        "rub" : 68.52
+    },
+    "conversions" : {
+        "<hash1>" : {
+            "eur" : 3.0
+        },
+        "<hash2>" : {
+            "timestamp" : 1507570800,
+            "eur" : 1.0,
+            "usd" : 1.175,
+            "czk" : 25.88,
+            "pln" : 4.303,
+            "rub" : 68.52
+        }
+    }
+}
+```
+
+
+### Contents: 
+ 
+ * [Part 1 - Setting up basic project](part1.md)
+ * [Part 2 - Sprinkle some types](part2.md)
+ * [Part 3 - Hello Kotlin](part3.md)
+ * [Part 4 - Even more Kotlin](part4.md)
+ * [Optional - A bit of auto-magic](part5.md)
+
+
+### Disclaimer:
+
+I am far from an expert in JavaScript, I dislike both languages to be completely honest. Therefore my JS code is very far from optimal and correct, but it still does it's job. Apologies to fans of this strange language.
```

Comment:
from being an
its job
the fans
both languages? JS and Kotlin? Did not know you disliked Kotlin 😭 

I like the disclaimer about your skill in the language, but I feel like your opinion on "liking" the language does not need to be here. Totally up to you tho.

I would tone this down a bit.
I am not a JavaScript expert, therefore the JS code here is far from optimal. Apologies to JS fans or experts that get eye-bleeds.